src/scripts/GoogleAnalyticsLoader.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @implements {GoogleAnalyticsLoader}
+ * @memberof module:GoogleAnalytics
```

Comment:
should i update update module name, for example:
```suggestion
 * @memberof module:Scripts
```